commands/bootstrap_cli.py

```diff
@@ -0,0 +1,269 @@
+#!/usr/bin/env python
+
+from collections import defaultdict
+from pathlib import Path
+
+import click
+from cloudfoundry_client.client import CloudFoundryClient
+from schema import Optional, Schema
+import yaml
+
+from .aws import get_ssm_secret_names, set_ssm_param, SSM_PATH, SSM_BASE_PATH
+from .templates import setup_templates
+from .utils import camel_case, mkdir, mkfile
+
+
+config_schema = Schema({
+    "app": str,
+    "environments": {
+        str: {
+            Optional("certificate_arns"): [str]
+        }
+    },
+    "services": [
+        {
+            "name": str,
+            "type": lambda s: s in ("public", "backend",),
+            "repo": str,
+            "image_location": str,
+            Optional("notes"): str,
+            Optional("secrets_from"): str,
+            "environments": {
+                str: {
+                    "paas": str,
+                    Optional("url"): str,
+                    Optional("ipfilter"): bool,
+                }
+            },
+            Optional("backing-services"): [
+                {
+                    "name": str,
+                    "type": lambda s: s in ("s3", "external-s3", "postgres", "redis", "opensearch",),
+                    Optional("paas-description"): str,
+                    Optional("paas-instance"): str,
+                    Optional("notes"): str,
+                    Optional("bucket_name"): str,           # for external-s3 type
+                    Optional("readonly"): bool,             # for external-s3 type
+                    Optional("shared"): bool,
+                }
+            ],
+            Optional("overlapping_secrets"): [ str ],
+            "secrets": {
+                Optional(str): str,
+            },
+            "env_vars": {
+                Optional(str): str,
+            },
+        }
+    ],
+})
+
+
+def get_paas_env_vars(client, paas):
+
+    org, space, app = paas.split("/")
+
+    env_vars = None
+
+    for paas_org in client.v2.organizations:
+        if paas_org["entity"]["name"] == org:
+            for paas_space in paas_org.spaces():
+                if paas_space["entity"]["name"] == space:
+                    for paas_app in paas_space.apps():
+                        if paas_app["entity"]["name"] == app:
+                            env_vars = paas_app["entity"]["environment_json"]
+
+    if not env_vars:
+        raise Exception(f"Application {paas} not found")
+
+    return dict(env_vars)
+
+
+def load_and_validate_config(path):
+
+    with open(path, "r") as fd:
+        conf = yaml.safe_load(fd)
+
+    # validate the file
+    schema = Schema(config_schema)
+    config = schema.validate(conf)
+
+    return config
+
+
+@click.group()
+def bootstrap():
+    pass
+
+
+@bootstrap.command()
+@click.argument("config-file", type=click.Path(exists=True))
+@click.argument("output", type=click.Path(exists=True), default=".")
+def make_config(config_file, output):
```

Comment:
This looks like it is not up to date with the `main` branch.

Reply:
This is now up to date 

The command is now

```
./copilot-helper.py bootstrap make-config storage.yaml
```

Reply:
Thank you kindly! Resolved in https://github.com/uktrade/copilot-tools/pull/31/commits/b54841bec25046a2cc34cd93679b7d9f55087f32.